### ISSUE TITLE
Enhancement Request

### ISSUE THREAD

**Issue by newgithubacc1v** [h-e-n-r-y/binance-marketmaker#4]
The following request might be too advanced for this project, but its worth a try. Maybe this can be part of a new project based on this project.

REQUEST:
Even though the project uses a basic strategy, this project seems very developed. One request would be to add a feature that monitors the "current" average price. For example, a user can set an option for the bot to take into consideration the average price of the last week (or) last couple of days (or) last couple of hours. Then the bot will be more independent.

EXAMPLE:
Currently, we could set a 1.30% difference of a price, but it is kind of based on luck.

With the feature, the bot will check the average weekly price (or) average daily price (or) average hourly price before each "buy/sell" or "sell/buy" cycle. Then it will adjust the price to better fit the average.

MORE ADVANCED REQUEST (NOT REQUIRED):
The user will be able to set whether the bot checks the average price each cycle or every other cycle, etc.

**Comment by h-e-n-r-y** [comment]
Hey,

Thanx for your contribution! 

Nice idea! 
In fact I also was thinking about this approach to give the bot the ability to follow rising and falling prices. Now it is only possible to follow linear price changes. And manual tweaking must be done regularly.

Another approach would be to take StochRSI into account for the right timing of placing an order....

Henry

**Comment by newgithubacc1v** [comment]
Yes, StochRSI would be even more accurate. I was trying to use a more basic approach, but if StochRSI is possible then it is probably better.
